e2e/tests/transfer/localhost_test.go

```diff
@@ -55,13 +55,15 @@ func (s *LocalhostTransferTestSuite) TestMsgTransfer_Localhost() {
 	t.Run("verify begin blocker was executed", func(t *testing.T) {
 		cs, err := s.QueryClientState(ctx, chainA, exported.LocalhostClientID)
 		s.Require().NoError(err)
-		originalHeight := cs.GetLatestHeight()
+
+		localhostClientState := cs.(*localhost.ClientState)
+		originalHeight := localhostClientState.LatestHeight
 
 		s.Require().NoError(test.WaitForBlocks(ctx, 1, chainA), "failed to wait for blocks")
 
 		cs, err = s.QueryClientState(ctx, chainA, exported.LocalhostClientID)
 		s.Require().NoError(err)
-		s.Require().True(cs.GetLatestHeight().GT(originalHeight), "client state height was not incremented")
+		s.Require().True(localhostClientState.LatestHeight.GT(originalHeight), "client state height was not incremented")
```

Comment:
this is going to be stale, needs to use `cs.LatestHeight` from the fresh query

```suggestion
		s.Require().True(cs.(*localhost.Client).LatestHeight.GT(originalHeight), "client state height was not incremented")
```